businessRules/ideas.js

```diff
@@ -38,34 +39,53 @@ function createIdeas(header, description, askedSum, userId) {
     createIdea(header, description, askedSum, userId);
 }
 
-function ideasStatusApproval(ideaId, approvalStatus) {
+function ideasStatusApproval(ideaId, isApproved) {
+    // Check if idea with this ideaId exists.
     const ideas = getAllIdeas();
     const idea = ideas.find((idea) => idea.id === ideaId);
 
-    if (approvalStatus === 'approved') {
-        updateIdea(ideaId, 'approve', 'approved');
-    } else {
-        deleteIdea(ideaId);
+    if (!idea) {
+        throw new Error(`Error. No idea with such ID found.`);
     }
+    // Idea exists.
+    isApproved
+        ? updateIdea(ideaId, 'approve', IDEA_STATUS.accepted)
```

Comment:
What does `'approve'` value represents here?

Reply:
Approve represents the key. I was thinking maybe it would be better to rename it tu "status" ?

Reply:
Updated. Commited. Tested.